cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityUserOperator.java

```diff
@@ -332,4 +332,25 @@ public void setContainerEnvVars(List<ContainerEnvVar> envVars) {
     public void setContainerSecurityContext(SecurityContext securityContext) {
         templateContainerSecurityContext = securityContext;
     }
+
+    /**
+     * Transforms map to log4j properties file format
+     * @param properties map with properties
+     * @return
+     */
+    @Override
+    protected String createPropertiesString(OrderedProperties properties) {
```

Comment:
These seem to be duplicates with the EnttityTopicOperator. Should we factor them out somewhere?